cherrypy/lib/auth_digest.py

```diff
@@ -417,7 +461,14 @@ def www_authenticate(
     )
 
 
-def digest_auth(realm, get_ha1, key, debug=False, accept_charset='utf-8'):
+def digest_auth(
+    realm,
+    get_ha1,
+    key,
+    algorithms=valid_algorithms,
```

Comment:
When modifying the public API signatures, always add new arguments at the end of the list for backward compatibility.